### PR TITLE
Refactors "strpos" calls in  lib/private to improve code readability.

### PR DESCRIPTION
**Pull request by fsamapoor**
- [x] Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [x] [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits

### CODE REVIEW

**Review by nfebe — APPROVED** [review]
LGTM

File: lib/private/Files/Type/Detection.php [hunk 1]
```diff
@@ -238,15 +238,15 @@ public function detectContent(string $path): string {
...
+				$mimeType = str_contains($info, ';') ? substr($info, 0, strpos($info, ';')) : $info;
 				$mimeType = $this->getSecureMimeType($mimeType);
 				if ($mimeType !== 'application/octet-stream') {
 					return $mimeType;
 				}
 			}
 		}
 
-		if (strpos($path, '://') !== false && strpos($path, 'file://') === 0) {
+		if (str_contains($path, '://') && str_starts_with($path, 'file://')) {
```
Comment (icewind1991) [inline lib/private/Files/Type/Detection.php:249]: these 2 can be combined as the latter implies the former

File: lib/private/TemplateLayout.php [hunk 2]
```diff
@@ -284,7 +284,7 @@ public function __construct($renderAs, $appId = '') {
 			} else {
 				$suffix = $this->getVersionHashSuffix($web, $file);
 
-				if (strpos($file, '?v=') == false) {
+				if (!strpos($file, '?v=')) {
```
Comment (icewind1991) [inline lib/private/TemplateLayout.php:287]: `!str_contains`

**Review by icewind1991 — APPROVED** [review]

File: lib/private/TemplateLayout.php (hunk 2 above)
Comment (fsamapoor) [reply lib/private/TemplateLayout.php]: Thanks. Done.

**Pull request merged**